script-menus.rpy

```diff
@@ -551,7 +584,7 @@ label datemenu:
             elif club:
```

Comment:
We should fix these too, just to be `if club`, `else`.

That said, we could do something better. Why use this convoluted system when we can simply use a global variable to store the current location? It's much easier than adding more vars to lose track of. (But that said I have another branch which a rewrite is happening in, so maybe it's best to not do that)

Reply:
I was thinking of using the persistent.date but I didn't wanna change too much in case it's used elsewhere so I just expanded the old system

Reply:
That's fine, pretty much all of the existing code is pending rewrite anyway so I'm not against not changing it